test/extended/testdata/builds/build-quota/.s2i/bin/assemble

```diff
@@ -4,8 +4,28 @@
 # output, so adding a sleep in an attempt to let the buildah log output
 # stop before the container output starts
 sleep 10
-echo -n MEMORY= && cat /sys/fs/cgroup/memory/memory.limit_in_bytes
-echo -n MEMORYSWAP= && cat /sys/fs/cgroup/memory/memory.memsw.limit_in_bytes
+cgroupv1Val=`cat /sys/fs/cgroup/memory/memory.limit_in_bytes` || true
+echo -n "cgroupv1Val is ${cgroupv1Val}"
+if [ $cgroupv1Val == "419430400" ]; then
```

Comment:
the default value is "max" in the cgroup file we access @adambkaplan and I saw that in the debug of my other tests

we need to check for that specific value to make sure the limit set on the BC is propagated

I saw similar results with cgroup v1 and what currently works with e2e-aws-builds

And if it helps at all, the check for that specific value has been around since 3.6 I believe and has not had to be changed

Reply:
Realized there was some imprecision in my last comment @adambkaplan .  Reattempting:
- prior forays with temp debug showed, now that we know where to look, the memory.max file is always there
- it always is non empty
- it can be a number or 'max' if no limits are set
- and yes, that precise number has not had to change since this was originally introduced back in 3.6

Aside from verifying that the builder image does the right interpretation, this test is also verifying that the OCM properly takes whatever limits the user set on the BC/build object and properly mapped those to the pod

Reply:
got it - let's keep it as this then since that file is always non-empty.